components/filter/events.tsx

```diff
@@ -26,28 +29,22 @@ interface EventsFilterProps {
 }
 
 export type EventTrimmed = {
-  HostedAtSSW?: boolean;
-  id: string;
-  Title: string;
-  Thumbnail: {
-    Url: string;
-    Description: string;
+  hostedAtSSW?: boolean;
+  id?: string;
+  title: string;
+  thumbnail?: {
+    url?: string;
+    description?: string;
   };
-  StartDateTime: Date;
-  EndDateTime: Date;
-  City: string;
-  Url: {
-    Url: string;
-    Description: string;
-  };
-  Presenter?: string;
-  PresenterProfileUrl?: {
-    Url: string;
-  };
-  CalendarType?: string;
-  // TODO: Fix the name of this field
-  Category_f5a9cf4c_x002d_8228_x00?: string;
-  EventShortDescription: string;
+  startDateTime: Date;
+  endDateTime: Date;
+  city?: string;
+  url: string;
+  presenter?: string;
```

Comment:
Assuming this is the presenter name, change to presenterName
```suggestion
  presenterName?: string;
```

Reply:
Done